### DISCUSSION TITLE
ModuleNotFoundError: No module named 'google.protobuf'

### DISCUSSION THREAD

**Discussion by Mr-Dj-Python** [google/or-tools#4194] [Build and configuration issues] [answered]
I have installed google and protobuf both library IDE(Pycharm) able to direct with correct library files and other things are visible but I it is not able to import during running example program provided in or tools documentation with error.

<img width="1512" alt="image" src="https://github.com/google/or-tools/assets/72373042/1a40f584-b729-4d5f-a655-0dcec8e599ac">

**Comment by lperron** [comment]
what happens if you have a single model that just imports google.protobuf (without or-tools) ?

**Reply by Mr-Dj-Python** [reply]
https://developers.google.com/optimization/routing/cvrp#complete_programs
I just started with the library so I am running the above example program.

<img width="1611" alt="image" src="https://github.com/google/or-tools/assets/72373042/6c7901ba-6998-4c3d-aaa4-87b01652f495">

Added pip freeze result one of my doubts is be using the wrong version but an error is raised from the library itself and if I try to import it example file it does raise same error.

<img width="1245" alt="image" src="https://github.com/google/or-tools/assets/72373042/97e4ff76-f387-4eac-b7e5-c1ac79c0fe45">

<img width="1096" alt="image" src="https://github.com/google/or-tools/assets/72373042/54261dc6-d4e0-49ef-b468-76aa5ca49fda">

**Reply by Mr-Dj-Python** [reply, marked as answer]
got problem there was another library named as google which cause the error
